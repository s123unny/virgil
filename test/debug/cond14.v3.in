b ../core/cond14.v3 25
b ../core/cond14.v3 31
b ../core/cond14.v3 32
run
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
q
