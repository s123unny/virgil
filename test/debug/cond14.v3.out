breakpoint|../core/cond14.v3 25|0
breakpoint|../core/cond14.v3 31|1
breakpoint|../core/cond14.v3 32|2
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|9
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|9
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|10
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|10
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|11
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|12
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|13
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|13
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|14
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|14
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|15
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|16
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|17
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|18
stop|stopOnBreakpoint
bt|mx|../core/cond14.v3|31
bt|main|../core/cond14.v3|21
stop|stopOnBreakpoint
bt|my|../core/cond14.v3|32
bt|main|../core/cond14.v3|22
stop|stopOnBreakpoint
bt|main|../core/cond14.v3|25
variable|0|a|false|bool|false
variable|1|b|false|bool|false
stop|end
result|8946
